llvm/lib/FuzzMutate/IRMutator.cpp

```diff
@@ -612,8 +612,8 @@ void ShuffleBlockStrategy::mutate(BasicBlock &BB, RandomIRBuilder &IB) {
     for (Instruction *Root : Roots)
       RS.sample(Root, 1);
     Instruction *Root = RS.getSelection();
-    Roots.erase(Root);
-    AliveInsts.erase(Root);
+    Roots.remove(Root);
+    AliveInsts.erase(std::find(AliveInsts.begin(), AliveInsts.end(), Root));
```

Comment:
You can just erase Root here.

Reply:
All overloads of erase require iterators (LLVM just uses T*), which should require find.